bindings/go/mcp/mcp.go

```diff
@@ -16,6 +16,8 @@
  * For any questions about this software or licensing,
  * please email opensource@seagate.com or cortx-questions@seagate.com.
  *
+ * Original author: Andriy Tkachuk <andriy.tkachuk@seagate.com>
+ * Original creation date: 30-Oct-2020
```

Comment:
All our source code does not have these fields any longer.

Reply:
I am not sure why the "authors" were removed from individual file.

Reply:
@madhavemuri , can you please comment on this?

Reply:
@mukul-seagate11 , what is your opinion?

Reply:
I do not have an opinion for this.

Reply:
It seems unnecessary since it shows up in 'blame' view and people can look at git logs.  But if people see value in it, I don't see any good reasons why we shouldn't allow it.

Reply:
OK